### PR TITLE
Add Rotated ROI align op for pytorch (cpu&cuda), parrots (cpu&cuda) and onnxruntime (cpu)

### PR DESCRIPTION
**Pull request by Johnson-Wang**
Awaiting steps. 
- [x] Add unittest for RoIAlignRotated

### CODE REVIEW

**Comment by ZwwWayne** [comment]
Please fix the CLA.

**Review by luopeichao — APPROVED** [review]
Parrots support is ready now.

File: mmcv/ops/csrc/onnxruntime/cpu/roi_align_rotated.cpp [hunk 1]
```diff
@@ -0,0 +1,246 @@
+#include "roi_align_rotated.h"
+
+#include "../ort_mmcv_utils.h"
+
+// Implementation taken from
```
Comment (ZwwWayne) [inline mmcv/ops/csrc/onnxruntime/cpu/roi_align_rotated.cpp:5]: Put it to the top of the file.

File: mmcv/ops/csrc/roi_align_rotated_cuda_kernel.cuh [hunk 2]
```diff
@@ -0,0 +1,199 @@
+#ifndef ROI_ALIGN_ROTATED_CUDA_KERNEL_CUH
```
Comment (ZwwWayne) [inline mmcv/ops/csrc/roi_align_rotated_cuda_kernel.cuh:1]: Should this file recognize https://github.com/facebookresearch/detectron2/blob/master/detectron2/layers/csrc/ROIAlignRotated/ROIAlignRotated_cpu.cpp?

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**